src/assets/logos/logo-with-text-white.svg

```diff
@@ -0,0 +1,108 @@
+	h9.092v-48.085h14.775v-7.694H332.054z M542.145,336.461c-2.885-2.797-6.032-4.896-9.704-6.295
+	c-3.585-1.486-7.432-2.186-11.453-2.186c-4.022,0-7.869,0.7-11.453,2.186c-3.585,1.486-6.82,3.585-9.705,6.295
+	c-2.885,2.798-4.983,5.945-6.47,9.53c-1.486,3.497-2.186,7.257-2.186,11.278c0,4.022,0.699,7.781,2.186,11.278
+	s3.672,6.645,6.47,9.442c2.798,2.71,6.032,4.896,9.705,6.382c3.672,1.486,7.431,2.186,11.453,2.186c3.934,0,7.78-0.699,11.365-2.186
+	s6.819-3.585,9.704-6.382c2.886-2.798,4.984-5.945,6.471-9.442s2.186-7.256,2.186-11.278c0-3.934-0.787-7.694-2.186-11.191
+	C547.128,342.494,544.942,339.259,542.145,336.461z M539.784,365.225c-1.05,2.536-2.536,4.809-4.459,6.732
+	c-1.924,2.011-4.196,3.497-6.645,4.546s-5.071,1.574-7.781,1.574c-2.798,0-5.421-0.524-7.868-1.574
+	c-2.448-1.049-4.634-2.536-6.558-4.546c-1.923-2.011-3.409-4.196-4.459-6.732c-1.049-2.536-1.486-5.158-1.486-8.043
+	c0-2.798,0.524-5.508,1.486-8.043c1.05-2.535,2.536-4.809,4.459-6.732c1.924-2.011,4.109-3.497,6.645-4.546
+	c2.448-1.049,5.071-1.574,7.869-1.574c2.71,0,5.333,0.525,7.868,1.574c2.448,1.049,4.634,2.536,6.557,4.546
+	c1.924,2.011,3.41,4.284,4.459,6.819c1.05,2.536,1.574,5.246,1.574,8.043C541.357,360.067,540.833,362.689,539.784,365.225z
+	 M580.176,359.63c4.197-0.525,7.432-2.011,9.617-4.546c2.186-2.536,3.235-6.033,3.235-10.492c0-2.535-0.438-4.896-1.399-6.994
+	s-2.272-3.847-4.021-5.246c-1.486-1.137-3.234-1.923-5.246-2.361c-2.011-0.437-4.896-0.699-8.655-0.699h-12.677v55.779h8.48v-25.004
+	h1.049l18.535,25.004h10.229L580.176,359.63z M570.122,352.985v-16h1.661c4.546,0,7.693,0.612,9.354,1.923
+	c1.661,1.224,2.535,3.41,2.535,6.47c0,2.885-0.786,4.809-2.36,5.945c-1.573,1.136-4.721,1.661-9.529,1.661H570.122z M452.88,385.071
+	h9.093v-26.229h22.119v-7.956h-22.119v-13.901h22.119v-7.694H452.88V385.071z M409.777,355.433c2.623-0.787,4.634-2.186,5.945-4.197
+	c1.399-2.011,2.099-4.458,2.099-7.431c0-2.448-0.438-4.721-1.399-6.819c-0.961-2.011-2.272-3.76-4.021-4.983
+	c-1.312-0.962-2.973-1.749-4.896-2.099c-1.924-0.437-4.896-0.612-8.918-0.612h-14.076v55.779h14.163
+	c5.334,0,9.093-0.35,11.366-0.961c2.273-0.612,4.109-1.749,5.683-3.235c1.486-1.399,2.535-3.147,3.322-5.158
+	c0.787-2.011,1.225-4.197,1.225-6.47c0-3.76-0.962-6.82-2.798-9.268C415.636,357.444,413.013,355.958,409.777,355.433z
+	 M393.429,336.811h3.497c4.634,0,7.693,0.612,9.354,1.836c1.661,1.224,2.448,3.41,2.448,6.47c0,2.797-0.874,4.809-2.71,5.945
+	c-1.836,1.224-4.896,1.749-9.181,1.749h-3.497v-15.999H393.429z M409.952,372.744c-0.611,1.224-1.486,2.273-2.622,2.972
+	c-0.962,0.612-2.187,1.049-3.76,1.312c-1.574,0.262-3.935,0.35-7.169,0.35h-2.973v-17.835h5.857c1.836,0,3.322,0.088,4.372,0.263
+	c1.136,0.175,2.098,0.524,2.972,0.961c1.399,0.7,2.448,1.749,3.147,3.06c0.787,1.312,1.137,2.885,1.137,4.634
+	C410.914,370.121,410.564,371.52,409.952,372.744z"/>
+<rect id="Line_1_" x="251.532" y="293.359" fill="#fff" width="6.294" height="205.37"/>
+<path id="Symbol_1_" fill="#fff" stroke="#9D4BA3" stroke-miterlimit="10" d="M216.036,369.597

```

Comment:
Change "stroke" to also be white. Right now there's still a purple outline.

Reply:
thanks!  I didn't notice the purple outline when i checked it